liquigraph-core/src/main/java/org/liquigraph/core/api/ClearChecksumRunner.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2014-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.api;
+
+import org.liquigraph.core.configuration.Configuration;
+import org.liquigraph.core.io.ClearChecksumWriter;
+import org.liquigraph.core.io.ConnectionSupplier;
+import org.liquigraph.core.io.GraphJdbcConnector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.function.Supplier;
+
+public class ClearChecksumRunner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClearChecksumRunner.class);
+
+    public void run(Configuration configuration) {
+        Supplier<Connection> connectionSupplier = new ConnectionSupplier(new GraphJdbcConnector(configuration));
```

Comment:
You can use `GraphJdbcConnector` and call `GraphJdbcConnector#connect` directly, you do not need to go through `ConnectionSupplier`

Reply:
done in commit eee1086c

I removed the class ConnectionSupplier and restored the previous codebase in its original state